### PR TITLE
add development environment requirements for development guide.

### PR DESCRIPTION
**Pull request by lijiangsheng1**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -19,14 +19,14 @@ branch, but release branches should not change.
...
-OpenPitrix only has few external dependencies you need to setup before being 
+OpenPitrix only has few external dependencies you need to setup before being
 able to build and run the code.
 
 ### Setting up Go
 
 OpenPitrix written in the [Go](http://golang.org) programming language.
-To build, you'll need a Go (version 1.8+) development environment. 
-If you haven't set up a Go development environment, please follow 
+To build, you'll need a Go (version 1.8+) development environment.
```
Comment (chai2010) [inline docs/development.md]: Go 1.9+

File: docs/development.md [hunk 2]
```diff
@@ -40,17 +40,22 @@ $ export PATH=$PATH:$GOPATH/bin
...
 ### Setting up Swagger
 
-OpenPitrix is using [OpenAPI/Swagger](https://swagger.io) to develop API, so follow 
-[the instructions](https://github.com/go-swagger/go-swagger/tree/master/docs) to 
-install Swagger. If you are not familar with Swagger, please read the 
-[tutorial](http://apihandyman.io/writing-openapi-swagger-specification-tutorial-part-1-introduction/#writing-openapi-fka-swagger-specification-tutorial). If you install Swagger using docker distribution, 
+OpenPitrix is using [OpenAPI/Swagger](https://swagger.io) to develop API, so follow
+[the instructions](https://github.com/go-swagger/go-swagger/tree/master/docs) to
+install Swagger. If you are not familar with Swagger, please read the
+[tutorial](http://apihandyman.io/writing-openapi-swagger-specification-tutorial-part-1-introduction/#writing-openapi-fka-swagger-specification-tutorial). If you install Swagger using docker distribution,
```
Comment (chai2010) [inline docs/development.md]: swagger is discard, we use grpc gateway now

File: docs/development.md (hunk 1 above)
Comment (rayzhou2017) [reply docs/development.md]: Go must be 1.9+?

Comment (chai2010) [reply docs/development.md]: make all will invoke go test, some test helper func requires Go 1.9. (t.Helper)

Comment (chai2010) [reply docs/development.md]: https://golang.org/doc/go1.9#test-helper

**Pull request merged**